emscripten.py

```diff
@@ -2519,16 +2518,12 @@ def create_sending_wasm(invoke_funcs, forwarded_json, metadata):
   if shared.Settings.SAFE_HEAP:
     basic_funcs += ['segfault', 'alignfault']
 
-  if not shared.Settings.RELOCATABLE:
-    global_vars = metadata['externs']
```

Comment:
What happened to externs? Are they not needed anymore?

Reply:
Not here (but I see they are used elsewhere). IIUC the externs are global variables, which we don't import from JS, we only import functions.